Description: Remove exception declarations
Author: Stephen Kitt <skitt@debian.org>

These were deprecated in C++11 and are no longer valid in C++17.

--- a/html/CharsetConverter.cc
+++ b/html/CharsetConverter.cc
@@ -7,7 +7,7 @@
 using namespace std;
 using namespace htmlcxx;
 
-CharsetConverter::CharsetConverter(const string &from, const string &to) throw (Exception)
+CharsetConverter::CharsetConverter(const string &from, const string &to)
 {
 	mIconvDescriptor = iconv_open(to.c_str(), from.c_str());
 	if (mIconvDescriptor == (iconv_t)(-1))
--- a/html/CharsetConverter.h
+++ b/html/CharsetConverter.h
@@ -17,7 +17,7 @@
 						: std::runtime_error(arg) {}
 			};
 			
-			CharsetConverter(const std::string &from, const std::string &to) throw (Exception);
+			CharsetConverter(const std::string &from, const std::string &to);
 			~CharsetConverter();
 			
 			std::string convert(const std::string &input);
